app/frontend/components/domains/permit-application/edit-permit-application-screen.tsx

```diff
@@ -330,7 +331,7 @@ export const EditPermitApplicationScreen = observer(({}: IEditPermitApplicationS
               gap={4}
               top={permitHeaderHeight}
             >
-              <Flex width="var(--app-sidebar-width)" align="center" gap={2}>
+              <Flex width={sizes.sidebar.width} align="center" gap={2}>
```

Comment:
@bkmaibach if you have added this to the sizes, you can directly use `width={"sidebar.width"}`